### PR TITLE
/download/intel-iei-tank-870 page refresh

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied refresh as per [design](https://www.figma.com/file/cVGimTPHky4h3BPXfrT6GB/24.04-U.com---download-(rebranding)?node-id=246%3A3249&mode=dev), [doc](https://docs.google.com/document/d/1Fa2JPxlEmZTyaFL_OSaQ-_OFW67AtBB2F-652TETqRk/edit), and [live page](https://ubuntu.com/download/intel-iei-tank-870)
  - Content should match the live page except where clear changes have been made on the copy doc (suggestion mode) 

## QA

- View the site locally in your web browser at: https://ubuntu-com-13576.demos.haus/download/intel-iei-tank-870
- Check against design, live page, and doc

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8797

### CODE REVIEW

**Comment by carkod** [comment]
Feature branch: [download-bubble-wd-8482](https://github.com/canonical/ubuntu.com/tree/download-bubble-wd-8482)

File: templates/download/intel-iei-tank-870.html [hunk 1]
```diff
@@ -89,19 +76,140 @@ <h3 class="p-stepped-list__title">
...
+        </div>
+      </div>
+      <hr />
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">Get started with snaps</h3>
+        </div>
+        <div class="col-6 col-medium-3">
+          <p>Your board is now ready to have snaps installed, it's time to use the snap command to install your first snap.</p>
+          <p>The <a href="https://snapcraft.io/store">Snap Store</a> is where you can find the best Linux apps packaged as snaps to install on your Ubuntu device and get started with your secure IoT journey.</p>
```
Comment (carkod) [inline templates/download/intel-iei-tank-870.html:112]: The copy doc has a link to `install your first snap`  https://docs.snapcraft.io/getting-started/3876

Comment (carkod) [reply templates/download/intel-iei-tank-870.html:112]: I had to update all the content in the copy doc, and then asked Juan to take another look at the copy doc for these kind of inconsistency issues...

File: templates/download/intel-iei-tank-870.html [hunk 2]
```diff
@@ -1,83 +1,70 @@
...
+          </div>
+        </li>
+        <li class="p-stepped-list__item">
+          <hr class="p-rule u-hide--large" />
+          <h2 class="p-stepped-list__title">Prepare the two USB flash drives</h2>
+          <div class="p-stepped-list__content">
+            <ol class="u-no-margin--left">
+              <li>Download and flash an <a href="/download/desktop">Ubuntu Desktop {{ releases.lts.full_version }} <abbr title="Long-term support">LTS</abbr></a> image on the first USB flash drive by following the live USB Ubuntu Desktop tutorial for <a href="/tutorials/tutorial-create-a-usb-stick-on-ubuntu">Ubuntu</a>, <a href="/tutorials/tutorial-create-a-usb-stick-on-windows">Windows</a>, or  <a href="/tutorials/tutorial-create-a-usb-stick-on-macos">macOS</a>.</li>
+              <li>Copy the Ubuntu Core image file to the second USB flash drive.</li>
+            </ol>
```
Comment (carkod) [inline templates/download/intel-iei-tank-870.html:49]: Shouldn't these have the separator? i.e. `p-list--divided` classes? but up to @lyubomir-popov

**Comment by lyubomir-popov** [comment]
@mtruj013 I've made some small changes +1 on my side, nice work!

**Review by carkod — APPROVED** [review]
A few comments for consistency. Else, LGTM.

**Comment by juanruitina** [comment]
LGTM!

File: templates/download/intel-iei-tank-870.html (hunk 1 above)
Comment (mtruj013) [reply templates/download/intel-iei-tank-870.html:112]: We were told that live content is the source of truth for these pages and this matches the copy on prod, so I'll leave it as is for now

File: templates/download/intel-iei-tank-870.html (hunk 2 above)
Comment (mtruj013) [reply templates/download/intel-iei-tank-870.html:49]: Oh I think you're right, numbered lists on the design do have that separator. He already left design +1 but I'll ping him to double check

**Pull request merged**